modules/host/persist.go

```diff
@@ -142,6 +177,11 @@ func (h *Host) establishDefaults() error {
 	}
 	h.spaceRemaining = h.settings.TotalStorage
 
+	// Reduce the window size when testing.
+	if build.Release == "testing" {
+		h.settings.WindowSize = testingWindowSize
+	}
```

Comment:
too much work for this PR, but in general I'm in favor of finding better ways to alter the constants for release vs dev vs testing.

Just don't want to think anymore tonight >.<